engine-src/Game/LambdaHack/Client/UI/Frontend/Sdl.hs

```diff
@@ -189,7 +189,9 @@ startupFun coscreen soptions@ClientOptions{..} rfMVar = do
   SDL.initialize [SDL.InitVideo]
   let screenV2 = SDL.V2 (toEnum $ rwidth coscreen * boxSize)
                         (toEnum $ rheight coscreen * boxSize)
-      windowConfig = SDL.defaultWindow {SDL.windowInitialSize = screenV2}
+      windowConfig = SDL.defaultWindow
+        { SDL.windowInitialSize = screenV2
+        , SDL.windowMode = if sfullscreen then SDL.Fullscreen else SDL.Windowed }
```

Comment:
OOI, what would `FullscreenDesktop` do here?

Reply:
That is  a "fake fullscreen" according to SDL wiki. So, I'm assuming, fullscreen windowed mode with no borders. We can add that as a separate config/CLI option, but I left it out here for simplicity.

Reply:
I've done some searching and it seems each of the modes don't work for some people. Do you think it would make sense to rename the option to windowMode and let the user specify any of the values that SDL accepts? The type has Read instance, so it should work be fine. In config, by default, Windowed could be set.

Reply:
Actually, it's easier to just define own 3-value type. Otherwise, each frontend module would need to provide a (usually erroring out) parser for the type, I think. Too complex.

Reply:
I can look into this. So you are saying you want me to add different CLI options for e.g. `--fullscreen` and `--fullscreen-windowed`?

Reply:
Thank you, I would be grateful. I guess I'd prefer to have the same schema in the config file and on command line. And in the config file, probably `windowMode = FullscreenDesktop` is natural and obvious. So `--windowMode FullscreenDesktop`.